### ISSUE TITLE
--blur and --saturate don't play well with --rotate

### ISSUE THREAD

**Issue by sector-f** [onespaceman/polyfy#11]
I noticed that if `--shape` is set to **triangle** and you use `--rotate`, it kinda breaks `--blur` and `--saturate`. Try it and see what I'm talking about. What I _think_ is happening is the matte (`/tmp/polyfy1.png`) is getting rotated properly but isn't getting moved to where the border is being drawn. Unfortunately, I'm not quite sure how to fix this.

**Comment by onespaceman** [comment]
Yeah. This is because the triangles height is different from its side length so the matte height != matte length (unlike with square, diamond, circle). Haven't had time to go back and figure out how to fix it, but it's on my todo list.
